src/Core/Grid/Definition/Factory/Security/Sessions/CustomerGridDefinitionFactory.php

```diff
@@ -0,0 +1,268 @@
+use PrestaShop\PrestaShop\Core\Grid\Column\ColumnCollection;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\ActionColumn;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\BulkActionColumn;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\DataColumn;
+use PrestaShop\PrestaShop\Core\Grid\Definition\Factory\AbstractGridDefinitionFactory;
+use PrestaShop\PrestaShop\Core\Grid\Definition\Factory\DeleteActionTrait;
+use PrestaShop\PrestaShop\Core\Grid\Filter\Filter;
+use PrestaShop\PrestaShop\Core\Grid\Filter\FilterCollection;
+use PrestaShop\PrestaShop\Core\Hook\HookDispatcherInterface;
+use PrestaShopBundle\Form\Admin\Type\SearchAndResetType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+/**
+ * Class CustomerGridDefinitionFactory responsible for creating Profiles grid definition.
+ */
+final class CustomerGridDefinitionFactory extends AbstractGridDefinitionFactory
+{
+    use DeleteActionTrait;
+
+    /**
+     * @var string
+     */
+    private $resetActionUrl;
+
+    /**
+     * @var string
+     */
+    private $redirectionUrl;
+
+    /**
+     * @param HookDispatcherInterface $hookDispatcher
+     * @param string $resetActionUrl
+     * @param string $redirectionUrl
+     * @param AccessibilityCheckerInterface $deleteProfileAccessibilityChecker
+     */
+    public function __construct(
+        HookDispatcherInterface $hookDispatcher,
+        $resetActionUrl,
+        $redirectionUrl
+    ) {
+        parent::__construct($hookDispatcher);
+        $this->resetActionUrl = $resetActionUrl;
+        $this->redirectionUrl = $redirectionUrl;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getId()
+    {
+        return 'security_sessions_customers';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getName()
+    {
+        return $this->trans('Customers sessions', [], 'Admin.Navigation.Menu');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getColumns()
+    {
+        return (new ColumnCollection())
+            ->add(
+                (new BulkActionColumn('bulk'))
+                    ->setOptions([
+                        'bulk_field' => 'id_customer_session',
+                    ])
+            )
+            ->add(
+                (new DataColumn('id_customer_session'))
+                    ->setName($this->trans('ID', [], 'Admin.Global'))
+                    ->setOptions([
+                        'field' => 'id_customer_session',
+                    ])
+            )
+            ->add(
+                (new DataColumn('id_customer'))
+                    ->setName($this->trans('Customer ID', [], 'Admin.Global'))
+                    ->setOptions([
+                        'field' => 'id_customer',
+                    ])
+            )
+            ->add(
+                (new DataColumn('firstname'))
+                    ->setName($this->trans('Firstname', [], 'Admin.Global'))
```

Comment:
I assume it is normal that there is no space between the 2 words?

Reply:
I don't understand the question? Which words are you talking about?

Reply:
Sorry, between "First" and "name"

Reply:
Done :)